nodejs/awsx/ecs/ec2Service.ts

```diff
@@ -154,6 +154,11 @@ export interface EC2TaskDefinitionArgs {
      */
     taskRole?: aws.iam.Role;
 
+    /**
+     * An optional family name for the Task Definition. If not specified, then a suitable default will be created.
+     */
+    family?: pulumi.Input<string>;
```

Comment:
can you do this for FargateTaskDefinitionArgs as well?

Reply:
Done! :)